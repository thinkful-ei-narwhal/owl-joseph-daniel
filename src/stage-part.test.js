import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import StagePart from './stage-part';


describe('<StagePart />', () => {
  it('smoke test', () => {
    const div = document.createElement('div')
    ReactDOM.render(<StagePart key={1} name={'Mike'} avatar={'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'} inSession={true} onStage={false}/>, div);
    // if something is wrong, itd fail here
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    const tree = renderer.create(<StagePart key={1} name={'Mike'} avatar={'https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1'} inSession={true} onStage={false}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});