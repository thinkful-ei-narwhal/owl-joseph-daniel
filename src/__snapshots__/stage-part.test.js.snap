// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StagePart /> renders this UI as expected 1`] = `
<div
  className="StagePart"
>
  <h3>
    Mike
  </h3>
  <img
    alt="Mike"
    src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
  />
</div>
`;
